src/main/java/net/pms/util/BufferedImageType.java

```diff
@@ -0,0 +1,244 @@
+/*
+ * Universal Media Server, for streaming any media to DLNA
+ * compatible renderers based on the http://www.ps3mediaserver.org.
+ * Copyright (C) 2012 UMS developers.
+ *
+ * This program is a free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; version 2
+ * of the License only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package net.pms.util;
+
+import java.awt.color.ColorSpace;
+import java.awt.image.BufferedImage;
+import java.awt.image.DirectColorModel;
+import java.awt.image.IndexColorModel;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This is an {@code enum} mirroring the integer constants in
+ * {@link BufferedImage}. Use this for better type-safety.
+ */
+public enum BufferedImageType {
+
+    /**
+     * Image type is not recognized so it must be a customized
+     * image.  This type is only used as a return value for the getType()
+     * method.
+     */
+    TYPE_CUSTOM(0),
+
+    /**
+     * Represents an image with 8-bit RGB color components packed into
+     * integer pixels.  The image has a {@link DirectColorModel} without
+     * alpha.
```

Comment:
Ah, I see what you mean now. I just copied those Javadocs from the [java.awt.image.BufferedImage](https://docs.oracle.com/javase/7/docs/api/java/awt/image/BufferedImage.html) source code. Please direct any complaints to Oracle 😛